generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx.ejs

```diff
@@ -51,9 +51,9 @@ export const UserManagement = (props: IUserManagementProps) => {
   useEffect(() => {
     const params = new URLSearchParams(props.location.search);
     const page = params.get('page');
-    const sort = params.get('sort');
-    if (page && sort) {
-      const sortSplit = sort.split(',');
+    const getSort = params.get('sort');
```

Comment:
is it a function or a regular primitive? (string, number, etc.)
here, I guess it's a function because it begins with a verb

Reply:
it's a string that I renamed because `sort` is already a function, but you're right it should not begin with a verb, I must find a name 😅

Reply:
this is one of the two most difficult things for a developper... cache invalidation & finding a good name ;)